### DISCUSSION TITLE
v2.1.0 - Maintenance Release

### DISCUSSION THREAD

**Discussion by iwalton3** [jellyfin/jellyfin-mpv-shim#269] [Announcements]
This release fixes some bugs and brings the software more up-to-date. I have done some testing against the newly released 10.8.0 server.

Changes:
- Upgrade to MPV version 20220515 9467772.
- Add flag for force_set_played due to double trakt play issue.
- Prevent multiple unwanted update checks.
- Fix to work with python-mpv 1.0 and MPV library v2.
- Add play_cmd and idle_ended_cmd. (#225)
- Allow disabling raise_mpv and also only do it once per play. (#218)
- Make transcode termination failure non-fatal. (#259)

About Windows Downloads:
 - You do not need administrator rights to install the software.
 - You can avoid warnings on files by unblocking the file in the properties dialog.
 - You can manually build a release [using these instructions](https://github.com/jellyfin/jellyfin-desktop#building-on-windows).

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin-mpv-shim/releases/tag/'>v2.1.0 - Maintenance Release</a>.</em>

**Comment by Dnkhatri** [comment]
Anyone else having the problem that mpv shim is not detected in browser instead need to run a 2nd instance before it is detected

**Reply by iwalton3** [reply]
Yes… https://github.com/jellyfin/jellyfin-mpv-shim/issues/268

**Reply by Dnkhatri** [reply]
Btw sorry I should have submitted this bug earlier but as was using with the 10.8 nighties I though no point had I submitted the regression might been caught before official release.
